nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees_desc_asb_[7,).partial.md

```diff
@@ -1,19 +1,44 @@
-In NServiceBus version 6 and above transports fully control the transactional support. A transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The Azure Service Bus transport only uses a transaction scope to implement the `SendAtomicWithReceive` transaction mode. Other modes work without using transaction scope. 
+When running in `SendAtomicWithReceive` mode, the Azure Service Bus (ASB) transport opens a transaction scope that cannot be enlisted by any other resource manager. As a result any custom behaviors or satellites need to explicitly wrap their transaction scope with a suppress scope.
 
-The new architecture for `SendAtomicWithReceive` is schematically represented in the diagram:
+This transaction scope is a requirement to allow the ASB transport to take advantage of, a little-known capability of the ASB SDK, [the via entity path / transfer queue](https://github.com/Azure-Samples/azure-servicebus-messaging-samples/tree/master/AtomicTransactions). This capability allows the transport to provide better delivery guarantees by reducing the number of duplicates sent out in case of exceptions occurring after the send operation.
+
+Using a transfer queue, send operations to different ASB entities can be executed via a single entity, the receive queue, and be completed in a single operation together with the acknowledgment that the receive operation has completed. Therefor messages are only sent when the receive operation completes and roll back if the receive operation is aborted. 
+
+Schematically it works like this:
+
+![Send Via](send-via.png)
+
+## Undesired side effects
+
+Transaction scopes automatically flow along the downstream execution path, and implicitly enlists any transactional resource in the ambient transaction.
```

Comment:
Seems to be a mismatch between "Transaction scopes" (plural) and "enlists" (singular).